src/dialog.c

```diff
@@ -42,6 +55,9 @@ void gtt_help_popup(GtkWidget *widget, gpointer data)
         gtk_widget_show(mb);
 
         printf("duude gnome help err msg: %s\n", err->message);
```

Comment:
Let's make this message a bit more professional while we're here. :) And a g_warning()?